VoiceOver/SoundQueueUI.lua

```diff
@@ -16,49 +16,131 @@ function SoundQueueUI:new(soundQueue)
     soundQueueUI:initNPCHead()
     soundQueueUI:initBookTexture()
     soundQueueUI:initControlButtons()
-
-    soundQueueUI.isDragging = false
+    soundQueueUI:initSettingsButton()
 
     return soundQueueUI
 end
 
 function SoundQueueUI:initDisplay()
-    self.soundQueueFrame = CreateFrame("Frame", nil, UIParent, "BackdropTemplate")
+    self.soundQueueFrame = CreateFrame("Frame", "VoiceOverSoundQueueFrame", UIParent, "BackdropTemplate")
     self.soundQueueScrollFrame = CreateFrame("ScrollFrame", nil, self.soundQueueFrame)
     self.soundQueueButtonContainer = CreateFrame("Frame", nil, self.soundQueueScrollFrame)
     self.soundQueueFrame:SetWidth(300)
     self.soundQueueFrame:SetHeight(300)
     self.soundQueueFrame:SetPoint("BOTTOMRIGHT", 0, 0)
     self.soundQueueFrame.buttons = {}
     self.soundQueueFrame:SetMovable(true)  -- Allow the frame to be moved
-    self.soundQueueFrame:EnableMouse(true) -- Allow the frame to be clicked on
+    self.soundQueueFrame:SetResizable(true)  -- Allow the frame to be resized
+    self.soundQueueFrame:SetClampedToScreen(true) -- Prevent from being dragged off-screen
+    self.soundQueueFrame:SetUserPlaced(true)
+    self.soundQueueFrame:SetResizeBounds(200, 40 + 64 + 10)
```

Comment:
1x Interface\AddOns\VoiceOver\SoundQueueUI.lua:36: attempt to call method 'SetResizeBounds' (a nil value)
[string "@Interface\AddOns\VoiceOver\SoundQueueUI.lua"]:36: in function `initDisplay'
[string "@Interface\AddOns\VoiceOver\SoundQueueUI.lua"]:15: in function `new'
[string "@Interface\AddOns\VoiceOver\SoundQueue.lua"]:9: in function `new'
[string "@Interface\AddOns\VoiceOver\Main.lua"]:5: in main chunk

Locals:
self = <table> {
 soundQueueScrollFrame = <unnamed> {
 }
 animtimer = 1680021736
 soundQueue = <table> {
 }
 soundQueueButtonContainer = <unnamed> {
 }
 soundQueueFrame = VoiceOverSoundQueueFrame {
 }
}
(*temporary) = nil
(*temporary) = VoiceOverSoundQueueFrame {
 0 = <userdata>
 OnBackdropLoaded = <function> defined @Interface\SharedXML\Backdrop.lua:144
 GetBackdropColor = <function> defined @Interface\SharedXML\Backdrop.lua:382
 SetupTextureCoordinates = <function> defined @Interface\SharedXML\Backdrop.lua:206
 OnBackdropSizeChanged = <function> defined @Interface\SharedXML\Backdrop.lua:174
 HasBackdropInfo = <function> defined @Interface\SharedXML\Backdrop.lua:270
 SetBackdropBorderColor = <function> defined @Interface\SharedXML\Backdrop.lua:414
 GetEdgeSize = <function> defined @Interface\SharedXML\Backdrop.lua:180
 buttons = <table> {
 }
 ApplyBackdrop = <function> defined @Interface\SharedXML\Backdrop.lua:286
 SetBackdrop = <function> defined @Interface\SharedXML\Backdrop.lua:321
 ClearBackdrop = <function> defined @Interface\SharedXML\Backdrop.lua:274
 GetBackdropBorderColor = <function> defined @Interface\SharedXML\Backdrop.lua:401
 GetBackdrop = <function> defined @Interface\SharedXML\Backdrop.lua:339
 SetupPieceVisuals = <function> defined @Interface\SharedXML\Backdrop.lua:238
 SetBackdropColor = <function> defined @Interface\SharedXML\Backdrop.lua:391
 SetBorderBlendMode = <function> defined @Interface\SharedXML\Backdrop.lua:258
}
(*temporary) = 200
(*temporary) = 114
(*temporary) = "attempt to call method 'SetResizeBounds' (a nil value)"

Reply:
1.14?

Reply:
Yep

Reply:
Should hopefully be fixed now